ioctl/cmd/hdwallet/hdwallet_test.go

```diff
@@ -86,7 +91,9 @@ func Test_Hdwallet(t *testing.T) {
 
 	private, err = wallet.PrivateKey(account)
 	require.NoError(err)
-	addr2, err := address.FromBytes(hashECDSAPublicKey(&private.PublicKey))
+	prvKey, err = crypto.BytesToPrivateKey(ecrypt.FromECDSA(private))
+	require.NoError(err)
+	addr2, err := address.FromBytes(prvKey.PublicKey().Hash())
```

Comment:
as i commented earlier, can directly r.Equal() private
```go
private2, err = wallet.PrivateKey(account)
require.NoError(err)
require.Equal(private, private2)
``` 
there's no need to do crypto.BytesToPrivateKey(), and address.FromBytes() in order to compare